CHANGELOG

```diff
@@ -1,6 +1,9 @@
 devel
 -----
 
+* The agency supervision is clearing "finished" jobs too quickly from
+  Target/Finished. They will be forthwith be kept for at least 5m.
```

Comment:
This should say 1h rather than 5min and it should mention `Target/Finished` as well as `Target/Failed`.

Reply:
Yes and mention `Failed` too.